kornia/geometry/transform/thin_plate_spline.py

```diff
@@ -79,19 +94,25 @@ def get_tps_transform(points_src: torch.Tensor, points_dst: torch.Tensor) -> tup
     pair_distance: torch.Tensor = _pair_square_euclidean(points_src, points_dst)
     k_matrix: torch.Tensor = _kernel_distance(pair_distance)
 
-    zero_mat: torch.Tensor = zeros(batch_size, 3, 3, device=device, dtype=dtype)
-    one_mat: torch.Tensor = ones(batch_size, num_points, 1, device=device, dtype=dtype)
+    zero_mat: torch.Tensor = torch.zeros(batch_size, 3, 3, device=device, dtype=dtype)
```

Comment:
```suggestion
    zero_mat: torch.Tensor = zeros(batch_size, 3, 3, device=device, dtype=dtype)
```
use as much as possible from kornia.core